viewing_party/party.py

```diff
@@ -1,23 +1,174 @@
+"""
+Dear Ashley (or Instructor),
+
+Can you please take a look at the comments in my wave 3.
+I tried doing something that didn't work out and so had to 
+abandon it all together. I can't figure out what the problem
+is and would appreciate a look.
+
+Also, answering my questions on Wednesday!
+And WOOT WOOT for finishing something I thought would even 
+be able to do halfway!!!!!
+
+Thank you so much,
+Huma Hameed
+(she/her)
+"""
+
+
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    for key, value in new_movie.items():
+        print(new_movie)
+        if value == None:
+            return None
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = []
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    # for movies["title"] in user_data["watched"] == title:
+    for movies in user_data["watchlist"]:
+        if movies["title"] == title:
+            print(title)

```

Comment:
Looks good! just remember to remove debugging print statements when you submit code.